dashboard/config/routes.rb

```diff
@@ -280,6 +280,8 @@
 
   get '/course/:course_name', to: redirect('/courses/%{course_name}')
   get '/courses/:course_name/vocab/edit', to: 'vocabularies#edit'
+  # this route uses course_course_name to match generated routes below that are nested within courses
+  get '/courses/:course_course_name/guides/edit', to: 'reference_guides#index'
```

Comment:
Should we save index for the user facing page? Maybe instead use something like `edit_all` ?

Reply:
done!